include/avif/avif.h

```diff
@@ -630,17 +630,14 @@ typedef struct avifGainMapMetadata
     //
     // If 'H' is the display's current log2-encoded HDR capacity (HDR to SDR ratio),
     // then the weight 'w' to apply the gain map is computed as follows:
-    // f = clamp((H - hdrCapacityMin) /
-    //           (hdrCapacityMax - hdrCapacityMin), 0, 1);
-    // w = backwardDirection ? f * -1 : f;
+    // f = clamp((H - baseHdrHeadroom) /
+    //           (alternateHdrHeadroom - baseHdrHeadroom), 0, 1);
+    // w = sign(alternateHdrHeadroom - baseHdrHeadroom) * f
     uint32_t baseHdrHeadroomN;
     uint32_t baseHdrHeadroomD;
     uint32_t alternateHdrHeadroomN;
     uint32_t alternateHdrHeadroomD;
 
-    // True if the gain map should be applied in reverse, see weight formula above.
-    avifBool backwardDirection;
```

Comment:
i know it's behind a flag, but may worth adding a CHANGELOG entry for this "public" ABI change.

Reply:
In this case it is not necessary to add a CHANGELOG entry, because the gain map code has not appeared in any libavif release yet.

Reply:
Added.

Reply:
Removed.